glean-core/src/upload/mod.rs

```diff
@@ -219,6 +225,16 @@ impl PingUploadManager {
         self.processed_pending_pings.load(Ordering::SeqCst)
     }
 
+    /// Gets the ping queue and checks in a ping with a certain document_id is already enqueued.
```

Comment:
```suggestion
    /// Checks if a ping with a certain `document_id` is already enqueued.
```

Reply:
I think you missed that. Your old phrasing has a typo in there and the function doesn't so much "gets the ping queue".
It just checks that something is in the queue it gets passed.

The shorter description captures that.